LPS-183473 - While in a Publication, entities deleted in Recycle Bin are deleted permanently

david-truong opened the pull request.

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**publications**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-source-format/4629/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/4629/jenkins-report.html">here</a>.</b></p>

david-truong commented:
@xbrianlee i think i reverted all tickets related to the recycle bin issue.  can you double check?  Also I added one last commit cause i still see this error in a customer's environment.

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee8d138b6c05dddf28b2b8548e17c505df2b6473">ee8d138b6c05dddf28b2b8548e17c505df2b6473</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/david-truong/liferay-portal/tree/LPS-183473">LPS-183473</a><br/>Branch GIT ID: <a href="https://github.com/david-truong/liferay-portal/commit/f7fc3e2433fe02013a9051652d38761b428566b3">f7fc3e2433fe02013a9051652d38761b428566b3</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/4629/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/4629/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/4629/">test-portal-source-format#4629</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/4629/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-publications/liferay-portal/pull/614">liferay-publications#614</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2352375822">[master] ci:test:sf - david-truong &gt; liferay-publications - PR#614 - 2023-06-15[09:25:01]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/17165/">publish-testray-report#17165</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/8716/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8716/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>publications</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/7475/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7475/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 28 out of 29 jobs passed</h3><h3>:x: ci:test:relevant - 106 out of 112 jobs passed in 3 hours 56 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee8d138b6c05dddf28b2b8548e17c505df2b6473">ee8d138b6c05dddf28b2b8548e17c505df2b6473</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee8d138b6c05dddf28b2b8548e17c505df2b6473">ee8d138b6c05dddf28b2b8548e17c505df2b6473</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2352439273">EE Development Acceptance (master) - 4616 - 2023-06-15[08:36:28]</a></p><details><summary><strong>ci:test:stable - 28 out of 29 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li>build-iw-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571241/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192229/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571246/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/526691/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/562969/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213429/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/182743/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/217535/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/212967/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/210865/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/230505/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/212586/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180628/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213431/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/223486/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/236151/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/215222/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/215219/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192227/">modules-compile-jdk8/0/11</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/536763/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/588743/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/620890/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192228/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/233286/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-mysql57-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608012/">portal-license-smoke-mysql57-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/233287/">rest-builder-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/208301/"><strike>service-builder-jdk8/0/0</strike></a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/433165/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/562968/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>28 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 106 out of 112 jobs PASSED</strong></summary><div><h4>6 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/8716/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571241/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192229/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-mysql57-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/440204/">empty-osgi-core-dir-mysql57-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-db2115-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/590061/">functional-smoke-tomcat90-db2115-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-hypersonic20-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/215221/">functional-smoke-tomcat90-hypersonic20-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/657409/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571246/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-tomcat90-oracle193-jdk8<ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180629/">functional-smoke-tomcat90-oracle193-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/216103/">functional-smoke-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-sqlserver2019-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/182739/">functional-smoke-tomcat90-sqlserver2019-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/220277/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/221376/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/620889/">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608010/">functional-tomcat90-mysql57-chrome100-jdk8/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/433163/">functional-tomcat90-mysql57-chrome100-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571244/">functional-tomcat90-mysql57-chrome100-jdk8/0/3</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/156662/">functional-tomcat90-mysql57-chrome100-jdk8/0/4</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/683662/">functional-tomcat90-mysql57-chrome100-jdk8/0/5</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/182744/">functional-tomcat90-mysql57-chrome100-jdk8/0/6</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571242/">functional-tomcat90-mysql57-chrome100-jdk8/0/7</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/433162/">functional-tomcat90-mysql57-chrome100-jdk8/0/8</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571243/">functional-tomcat90-mysql57-chrome100-jdk8/0/9</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/562971/">functional-tomcat90-mysql57-chrome100-jdk8/0/10</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/683663/">functional-tomcat90-mysql57-chrome100-jdk8/0/11</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/433164/">functional-tomcat90-mysql57-chrome100-jdk8/0/12</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/571245/">functional-tomcat90-mysql57-chrome100-jdk8/0/13</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/539504/">functional-tomcat90-mysql57-chrome100-jdk8/0/14</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/536764/">functional-tomcat90-mysql57-chrome100-jdk8/0/15</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/588744/">functional-tomcat90-mysql57-chrome100-jdk8/0/16</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/620891/">functional-tomcat90-mysql57-chrome100-jdk8/0/17</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608011/">functional-tomcat90-mysql57-chrome100-jdk8/0/18</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/156663/">functional-tomcat90-mysql57-chrome100-jdk8/0/19</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/657411/">functional-tomcat90-mysql57-chrome100-jdk8/0/20</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/539503/">functional-tomcat90-mysql57-chrome100-jdk8/0/21</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/566906/">functional-tomcat90-mysql57-chrome100-jdk8/0/22</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/526692/">functional-tomcat90-mysql57-chrome100-jdk8/0/23</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/562970/">functional-tomcat90-mysql57-chrome100-jdk8/0/24</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/590060/">functional-tomcat90-mysql57-chrome100-jdk8/1/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/566905/">functional-tomcat90-mysql57-chrome100-jdk8/1/1</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/210864/">functional-tomcat90-mysql57-chrome100-jdk8/1/2</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/156660/">functional-tomcat90-mysql57-chrome100-jdk8/1/3</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/182741/">functional-tomcat90-mysql57-chrome100-jdk8/1/4</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/657408/">functional-tomcat90-mysql57-chrome100-jdk8/1/5</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/536760/">functional-tomcat90-mysql57-chrome100-jdk8/1/6</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/590059/">functional-tomcat90-mysql57-chrome100-jdk8/1/7</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/566904/">functional-tomcat90-mysql57-chrome100-jdk8/1/8</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/182742/">functional-tomcat90-mysql57-chrome100-jdk8/1/9</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/526689/">functional-tomcat90-mysql57-chrome100-jdk8/1/10</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/215220/">functional-tomcat90-mysql57-chrome100-jdk8/1/11</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192226/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/1/12</strike></a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/590058/">functional-tomcat90-mysql57-chrome100-jdk8/1/13</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/182740/">functional-tomcat90-mysql57-chrome100-jdk8/1/14</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/440202/">functional-tomcat90-mysql57-chrome100-jdk8/1/15</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/526690/">functional-tomcat90-mysql57-chrome100-jdk8/1/16</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/588741/">functional-tomcat90-mysql57-chrome100-jdk8/1/17</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/536761/">functional-tomcat90-mysql57-chrome100-jdk8/1/18</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/440201/">functional-tomcat90-mysql57-chrome100-jdk8/1/19</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/236150/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/1/20</strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/233387/">functional-tomcat90-mysql57-chrome100-jdk8/1/21</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608009/">functional-tomcat90-mysql57-chrome100-jdk8/1/22</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/156661/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/208300/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/588742/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/536762/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/440203/">functional-upgrade-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/657410/">functional-upgrade-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/230504/">functional-upgrade-tomcat90-mysql57-jdk8/0/6</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/526691/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/562969/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213429/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/182743/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/220276/">functional-upgrade-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>lpkg-controller-mysql57-jdk8<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/221377/">lpkg-controller-mysql57-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/217535/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/212967/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/210865/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/230505/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/212586/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/180628/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213431/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/223486/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/236151/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/215222/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/215219/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192227/">modules-compile-jdk8/0/11</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/566907/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/440205/"><strike>modules-integration-mysql57-jdk8/0/1</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/657413/">modules-integration-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/657412/">modules-integration-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/536765/">modules-integration-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/588745/">modules-integration-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/590062/">modules-integration-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/156664/">modules-integration-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/539505/">modules-integration-mysql57-jdk8/0/8</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/536763/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/588743/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/620890/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/562972/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/233388/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192228/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/208099/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224741/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-mysql57-jdk8<ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/213430/">patching-tool-mysql57-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/233286/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-mysql57-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608012/">portal-license-smoke-mysql57-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/233287/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/683664/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/208301/"><strike>service-builder-jdk8/0/0</strike></a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/212585/"><strike>unit-jdk8/0/0</strike></a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/433165/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/562968/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>106 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8716/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/440205//consoleText">modules-integration-mysql57-jdk8/0/1</a><pre>[beanshell] Date:   Tue Oct 20 11:50:51 2020 -0700
[beanshell] 
[beanshell]     LPS-65627 Release
[beanshell] 
[beanshell] 
[beanshell] commit f9c7639200493cbb0288f1332b015ea8166f7350
[beanshell] Author: Neil Griffin &lt;neil.griffin.scm@gmail.com&gt;
[beanshell] Date:   Wed Apr 17 15:31:21 2019 -0400
[beanshell] 
[beanshell]     [maven-release-plugin] prepare release pluto-3.1.0
[beanshell] 
[beanshell] Executing commands: git branch | grep \*
[beanshell] Executing commands: git clean -dfx
[beanshell] Executing commands: git for-each-ref refs/heads --format="%(refname)"
[beanshell] Executing commands: git clean -dfx
[beanshell] Executing commands: git log -n 1
[beanshell] Executing commands: git clean -dfx
[beanshell] Executing commands: git branch | grep \*
[beanshell] 
[beanshell] commit 86e76ed8b8a561ea5908851a31a6e8b9e7798b9c
[beanshell] Author: Liferay Continuous Integration &lt;continuous-integration@liferay.com&gt;
[beanshell] Date:   Thu Jun 15 02:44:03 2023 -0700
[beanshell] 
[beanshell]     Commit cache
[beanshell] 
[beanshell] Executing commands: git branch | grep \*
[beanshell] Executing commands: git for-each-ref refs/heads --format="%(refname)"
[beanshell] Executing commands: git for-each-ref refs/heads --format="%(refname)"
[beanshell] Executing commands: git log -n 1
[beanshell] Executing commands: git log -n 1
[beanshell] Executing commands: git branch | grep \*
[beanshell] 
[beanshell] commit abad28b6c770a2ea7ccfb9d42f7d92367d486878
[beanshell] Author: Liferay Translation &lt;lta@liferay.com&gt;
[beanshell] Date:   Thu Jun 15 14:44:36 2023 +0000
[beanshell] 
[beanshell]     LPS-77699 Update Translations
[beanshell] 
[beanshell] 
[beanshell] commit 47b89877c140ec465f4488395f93aaf9dc823f5a
[beanshell] Author: Vinicius Lopes &lt;vinicius.lopes@liferay.com&gt;
[beanshell] Date:   Thu Mar 2 16:46:34 2023 -0300
[beanshell] 
[beanshell]     LRAC-12922 Implement logic to check if the chart has data for the QA team
[beanshell] 
[beanshell] Executing commands: git for-each-ref refs/heads --format="%(refname)"
[beanshell] Executing commands: git log -n 1
[beanshell] 
[beanshell] commit ee8d138b6c05dddf28b2b8548e17c505df2b6473
[beanshell] Author: Ambrin Chaudhary &lt;ambrin.chaudhary@liferay.com&gt;
[beanshell] Date:   Wed Jun 14 11:18:53 2023 +0200
[beanshell] 
[beanshell]     LPS-168067 Change missing modal header
[beanshell] 
[stopwatch] [stop.current.job: 44.583 sec]

BUILD SUCCESSFUL
Total time: 51 seconds
+ echo 'Exit shell.'
Exit shell.
Finished: FAILURE</pre></div></li><li><div><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/208301//consoleText">service-builder-jdk8/0/0</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/liferay-publications/liferay-portal/pull/614">david-truong/LPS-183473</a></strong><pre>     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.object.exception                       MINOR      18.3.0     18.3.0     18.4.0     VERSION INCREASE REQUIRED
     [exec]   +   class      com.liferay.object.exception.ObjectDefinitionNameException\$ForbiddenModifiableSystemObjectDefinitionName
     [exec]     +   access     static
     [exec]     +   extends    com.liferay.object.exception.ObjectDefinitionNameException
     [exec]     +   extends    com.liferay.portal.kernel.exception.PortalException
     [exec]     +   method     &lt;init&gt;(java.lang.String)
     [exec]       +   return     void
     [exec]     +   method     clone()
     [exec]       +   access     protected
     [exec]       +   return     java.lang.Object
     [exec]     +   method     equals(java.lang.Object)
     [exec]       +   return     boolean
     [exec]     +   method     finalize()
     [exec]       +   access     protected
     [exec]       +   return     void
     [exec]     +   method     hashCode()
     [exec]       +   return     int
     [exec]     +   method     toString()
     [exec]       +   return     java.lang.String
     [exec] [Baseline Warning] Bundle Version Change Recommended: 69.1.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.object.api-69.0.1.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.object.api/69.0.0/94e383596e871f764bf6fe2d6c00abf1b4895989/com.liferay.object.api-69.0.0.jar</pre></p></div></div></li><li><div><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192226//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/1/12</a><ol><li><div><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/192226//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PublicationsConflict_ResolveDeletionModificationConflictPublicationsByDiscarding_">LocalFile.PublicationsConflict#ResolveDeletionModificationConflictPublicationsByDiscarding</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-27/test-portal-acceptance-pullrequest(master)/8716/functional-tomcat90-mysql57-chrome100-jdk8/1/12/LocalFile.PublicationsConflict_ResolveDeletionModificationConflictPublicationsByDiscarding/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-27/test-portal-acceptance-pullrequest(master)/8716/functional-tomcat90-mysql57-chrome100-jdk8/1/12/LocalFile.PublicationsConflict_ResolveDeletionModificationConflictPublicationsByDiscarding/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-27/test-portal-acceptance-pullrequest(master)/8716/functional-tomcat90-mysql57-chrome100-jdk8/1/12/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2351931975/history">Failed 1 of last 23</a></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at ee8d138b6c05dddf28b2b8548e17c505df2b6473:</strong></summary><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/236150//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/1/20</a><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/236150//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_StagingUsecase_TrashEntryReferencePublishing_">LocalFile.StagingUsecase#TrashEntryReferencePublishing</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-27/test-portal-acceptance-pullrequest(master)/8716/functional-tomcat90-mysql57-chrome100-jdk8/1/20/LocalFile.StagingUsecase_TrashEntryReferencePublishing/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-27/test-portal-acceptance-pullrequest(master)/8716/functional-tomcat90-mysql57-chrome100-jdk8/1/20/LocalFile.StagingUsecase_TrashEntryReferencePublishing/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-27/test-portal-acceptance-pullrequest(master)/8716/functional-tomcat90-mysql57-chrome100-jdk8/1/20/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2351936869/history">Failed 12 of last 23</a></div></li></ol></div></li><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/212585//consoleText">unit-jdk8/0/0</a><ol><li><div>com.liferay.taglib.util.HtmlTopTagTest.testDataSennaTrackAttribute - UNTESTED</div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/8716/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/8716/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/8716/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/7475/">test-portal-acceptance-pullrequest(master)#7475</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7475/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>publications<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-publications/liferay-portal/pull/614">liferay-publications#614</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2352563281">[master] ci:test:publications - david-truong &gt; liferay-publications - PR#614 - 2023-06-15[10:40:08]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/17175/">publish-testray-report#17175</a><br/></div></div>

xbrianlee commented:
Continued in https://github.com/xbrianlee/liferay-portal/pull/1239
